src/test/java/uk/gov/pay/directdebit/tasks/resources/ExpireResourceTest.java

```diff
@@ -55,5 +55,22 @@ public void shouldExpireATransactionInStateStartedOlderThanOneDay() throws Excep
             .body("numberOfExpiredPayments", is(1));
     }
 
-
+    @Test
+    public void shouldExpireAMandateInState_CREATED_OlderThan_90Minutes() throws Exception {
+        MandateFixture.aMandateFixture()
+                .withState(MandateState.CREATED)
+                .withCreatedDate(ZonedDateTime.now().minusMinutes(91L))
+                .withGatewayAccountFixture(testGatewayAccount)
+                .insert(testContext.getJdbi());
+        
+        String requestPath = "/v1/api/tasks/expire-payments-and-mandates";
+        given()
+            .port(testContext.getPort())
+            .contentType(JSON)
+            .post(requestPath)
+            .then()
+            .statusCode(Response.Status.OK.getStatusCode())
+            .contentType(JSON)
+            .body("numberOfExpiredMandates", is(1));
```

Comment:
do we need to check the DB too if it's actually deleted?

Reply:
Good point, I've added a check to ensure the mandate status is set to EXPIRED.